browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -103,8 +118,22 @@ void BraveTabStrip::AddedToWidget() {
   }
 }
 
+SkColor BraveTabStrip::GetTabBackgroundColor(
+    TabActive active,
+    BrowserFrameActiveState active_state) const {
+  if (!ShouldShowVerticalTabs())
+    return TabStrip::GetTabBackgroundColor(active, active_state);
+
+  const ui::ColorProvider* cp = GetColorProvider();
+  if (!cp)
+    return gfx::kPlaceholderColor;
+
+  return cp->GetColor(active == TabActive::kActive ? kColorLocationBarBackground
```

Comment:
I'm wondering if we make a new color name specifically for vertical tabs because it seems a coincidence that it uses the same as the Location Bar background. Changing LocationBar background would change this and it might be considered a bug.

Reply:
Done!